src/Content/Conversation.php

```diff
@@ -1084,6 +1091,31 @@ private function getEmojis(array $uriids): array
 		return $emojis;
 	}
 
+	/**
+	 * Fetch quote shares from the conversation
+	 *
+	 * @param array $uriids
+	 * @return array
+	 */
+	private function getQuoteShares(array $uriids): array
+	{
+		$condition = DBA::mergeConditions(['quote-uri-id' => $uriids], ["NOT `quote-uri-id` IS NULL"]);
+		$separator = chr(255) . chr(255) . chr(255);
```

Comment:
What is this string?

Reply:
This is the same functionality like in `getEmojis`. We need to split the author names and need some separator for that.

Reply:
What character is `chr(255)`? Is it ever used for display?

Reply:
It is only used for the old DOS code pages. There it stands for the [nbsp](https://en.wikipedia.org/wiki/Non-breaking_space#Encodings).